<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>676</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>-10</y>
      <width>791</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	background-color: rgb(255, 255, 255);

}</string>
    </property>
    <widget class="QWidget" name="equipo_page">
     <property name="styleSheet">
      <string notr="true">background-color: #fff
</string>
     </property>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>10</y>
        <width>801</width>
        <height>701</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(225, 233, 243);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="guardarmodeloBoton_2">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>500</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font:  bold 10pt &quot;Century Gothic&quot;;
background-color: rgb(241, 243, 244);

}
QPushButton:hover{	
background-color: rgb(184, 191, 199);
}
</string>
       </property>
       <property name="text">
        <string>Guardar Modelo</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame_5">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>160</y>
         <width>341</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
border: 0.5px solid #000000;
border-radius: 5px;
background-color: rgb(255, 255, 255);

}
QFrame:hover {
border:  0.5px solid #567CF0; 
 }

QFrame::hover {
border: 0.5px solid #567CF0;    
}

                            
QFrame:focus {
border:  0.5px solid #567CF0;     
 }

QFrame:focus-within {
 border: 0.5px solid #567CF0;
}

 QFrame {
 box-shadow: 5px 5px 5px 5px #888888;}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="avscoretext">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="saldoframe">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>160</y>
         <width>261</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
border: 0.5px solid #000000;
border-radius: 5px;
	background-color: rgb(255, 255, 255);

}
QFrame:hover {
border:  0.5px solid #567CF0; 
 }

QFrame::hover {
border: 0.5px solid #567CF0;    
}

                            
QFrame:focus {
border:  0.5px solid #567CF0;     
 }

QFrame:focus-within {
 border: 0.5px solid #567CF0;
}

 QFrame {
 box-shadow: 5px 5px 5px 5px #888888;}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="saldotext">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="framebienvenido">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>451</width>
         <height>49</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(225, 233, 243);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="bienvenido">
          <property name="font">
           <font>
            <family>Century Gothic</family>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame{
	font: bold 15pt &quot;Century Gothic&quot;;
}</string>
          </property>
          <property name="text">
           <string>Bienvenido/a, </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="nombreUsuario">
          <property name="font">
           <font>
            <family>Century Gothic</family>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame{
	font: bold 15pt &quot;Century Gothic&quot;;
}</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_4">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>330</y>
         <width>661</width>
         <height>261</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
border: 0.5px solid #000000;
border-radius: 5px;
	background-color: rgb(255, 255, 255);
	border-radius: 3px;
  box-shadow: 0px 5px 15px 0px rgba(0, 0, 0, 0.52);

}
QFrame:hover {
border:  0.5px solid #567CF0; 
 }

QFrame::hover {
border: 0.5px solid #567CF0;    
}

                            
QFrame:focus {
border:  0.5px solid #567CF0;     
 }

QFrame:focus-within {
 border: 0.5px solid #567CF0;
}

 QFrame {
 box-shadow: 5px 5px 5px 5px #888888;

}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QLabel" name="bienvenido_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>130</y>
         <width>141</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
	font: bold 12pt &quot;Century Gothic&quot;;
}</string>
       </property>
       <property name="text">
        <string>Presupuesto </string>
       </property>
      </widget>
      <widget class="QLabel" name="bienvenido_4">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>130</y>
         <width>231</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
	font: bold 12pt &quot;Century Gothic&quot;;
}</string>
       </property>
       <property name="text">
        <string>Puntuaci√≥n Media</string>
       </property>
      </widget>
      <widget class="QLabel" name="bienvenido_5">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>290</y>
         <width>161</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
	font: bold 12pt &quot;Century Gothic&quot;;
}</string>
       </property>
       <property name="text">
        <string>Tus jugadores</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>290</y>
         <width>221</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox{
font:  bold 10pt &quot;Century Gothic&quot;;
background-color: rgb(241, 243, 244);
text-align: center;

}</string>
       </property>
       <item>
        <property name="text">
         <string>Equipo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Porteros</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Defensas</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mediocentros</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Delanteros</string>
        </property>
       </item>
      </widget>
      <zorder>guardarmodeloBoton_2</zorder>
      <zorder>saldoframe</zorder>
      <zorder>frame_5</zorder>
      <zorder>framebienvenido</zorder>
      <zorder>frame_4</zorder>
      <zorder>bienvenido_3</zorder>
      <zorder>bienvenido_4</zorder>
      <zorder>bienvenido_5</zorder>
      <zorder>comboBox</zorder>
     </widget>
    </widget>
    <widget class="QWidget" name="predicciones_page">
     <property name="styleSheet">
      <string notr="true">Qframe{
	background-color: rgb(255, 255, 255);


}</string>
     </property>
     <widget class="QFrame" name="recomendacioframe">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>180</y>
        <width>371</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
border: 0.5px solid #000000;
border-radius: 5px;


}
QFrame:hover {
border:  0.5px solid #567CF0; 
 }

QFrame::hover {
border: 0.5px solid #567CF0;    
}

                            
QFrame:focus {
border:  0.5px solid #567CF0;     
 }

QFrame:focus-within {
 border: 0.5px solid #567CF0;
}

 QFrame {
 box-shadow: 5px 5px 5px 5px #888888;}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="mercadoframe">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>281</width>
        <height>461</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
border: 0.5px solid #000000;
border-radius: 5px;


}
QFrame:hover {
border:  0.5px solid #567CF0; 
 }

QFrame::hover {
border: 0.5px solid #567CF0;    
}

                            
QFrame:focus {
border:  0.5px solid #567CF0;     
 }

QFrame:focus-within {
 border: 0.5px solid #567CF0;
}

 QFrame {
 box-shadow: 5px 5px 5px 5px #888888;}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="framebienvenido_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>711</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(225, 233, 243);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="bienvenido_2">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
	font: bold 18pt &quot;Century Gothic&quot;;
}</string>
         </property>
         <property name="text">
          <string>Consigue los mejores jugadores para tu equipo</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>10</y>
        <width>781</width>
        <height>701</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(225, 233, 243);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="scaperboton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>120</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: bold 11pt  &quot;Century Gothic&quot;;
	background-color: rgb(241, 243, 244);



}
QPushButton:hover{	
background-color: rgb(184, 191, 199);
}</string>
       </property>
       <property name="text">
        <string>Cargar jugadores del mercado</string>
       </property>
      </widget>
      <widget class="QComboBox" name="algoritmobox">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>120</y>
         <width>361</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox{
font:  bold 10pt &quot;Century Gothic&quot;;
background-color: rgb(241, 243, 244);

}</string>
       </property>
       <item>
        <property name="text">
         <string>Algoritmo 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Algoritmo 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Algoritmo 3</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="guardarmodeloBoton">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>500</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font:  bold 10pt &quot;Century Gothic&quot;;
background-color: rgb(241, 243, 244);

}
QPushButton:hover{	
background-color: rgb(184, 191, 199);
}
</string>
       </property>
       <property name="text">
        <string>Guardar Modelo</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>440</y>
         <width>381</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QFrame" name="modeloframe">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>470</y>
         <width>241</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
border: 0.5px solid #000000;
border-radius: 5px;
background-color: rgb(255, 255, 255);


}
QFrame:hover {
border:  0.5px solid #567CF0; 
 }

QFrame::hover {
border: 0.5px solid #567CF0;    
}

                            
QFrame:focus {
border:  0.5px solid #567CF0;     
 }

QFrame:focus-within {
 border: 0.5px solid #567CF0;
}

 QFrame {
 box-shadow: 5px 5px 5px 5px #888888;}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </widget>
     <zorder>frame</zorder>
     <zorder>recomendacioframe</zorder>
     <zorder>mercadoframe</zorder>
     <zorder>framebienvenido_2</zorder>
    </widget>
   </widget>
   <widget class="QFrame" name="subevntanas">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>211</width>
      <height>781</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(198, 213, 245);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="menu">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>211</width>
       <height>711</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #4d82bc;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="columnaMenu">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>201</width>
        <height>351</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>1000</number>
      </property>
      <property name="midLineWidth">
       <number>1000</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="botonequipo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
font: bold 11pt  &quot;Century Gothic&quot;;
background-color: rgb(241, 243, 244);
}

QPushButton:hover{	
background-color: #c4dafa;
}
</string>
         </property>
         <property name="text">
          <string>TU EQUIPO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botonpredicciones">
         <property name="styleSheet">
          <string notr="true">QPushButton{
font: bold 11pt  &quot;Century Gothic&quot;;
background-color: rgb(241, 243, 244);
}
QPushButton:hover{	
background-color: #c4dafa;
}
</string>
         </property>
         <property name="text">
          <string>PREDICCIONES</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="framesuperior">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>201</width>
        <height>181</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frametitulo">
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="titulo">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>-10</y>
            <width>201</width>
            <height>51</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>191</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Century Gothic</family>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame{
font:  bold 9pt &quot;Century Gothic&quot;;
color: rgb(255, 255, 255);
}</string>
          </property>
          <property name="text">
           <string>Dream League Assistant</string>
          </property>
         </widget>
         <widget class="QFrame" name="framelogo">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>50</y>
            <width>179</width>
            <height>76</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="logo">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>0</y>
             <width>81</width>
             <height>81</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/logo_dla.qrc/interfaz/logo_dla-removebg-preview (3).png);
background-color: transparent;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>logo_dla-removebg-preview (3).png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
